### DISCUSSION TITLE
Button to close all open windows at once

### DISCUSSION THREAD

**Discussion by BeanXOfficial** [f1multiviewer/issue-tracker#31] [Feature requests] [answered]
After testing the application a bit more, I think it would be rather useful having a button to close all opened windows at once.

**Comment by f1multiviewer** [comment, marked as answer]
<img width="248" alt="image" src="https://user-images.githubusercontent.com/106821994/178108192-3e458a9f-91b7-4738-bc79-71873ae3f042.png">
This should do 🤞

**Reply by timespacedecay** [reply]
That only closes the videos, it does not close live timing, track map, race trace, etc.
